### DISCUSSION TITLE
Symbolic folder links and folder_paths.py

### DISCUSSION THREAD

**Discussion by iyiarz** [comfyanonymous/ComfyUI#1012] [General]
I created a symbolic folder link (mklink /D) so I could share my checkpoints with Automatic 1111, rather than having multiple copies. It seems the script to check the folders doesn't support symbolic links and it deleted and recreated the folder instead.

Perhaps the script can be updated? I believe there's a function is_symlink() ?

**Comment by lord-lethris** [comment]
I use symbolic links - its fine for me. - what OS are you using?

**Comment by dishuostec** [comment]
There is a [`extra_model_paths.yaml.example`](https://github.com/comfyanonymous/ComfyUI/blob/master/extra_model_paths.yaml.example) file in project root, you can add extra search path.
